source/class/qx/tool/cli/api/CompilerApi.js

```diff
@@ -56,7 +56,7 @@ qx.Class.define("qx.tool.cli.api.CompilerApi", {
     
     /**
      * Register compiler tests
-     * @param  cmd: {qx.tool.cli.commands.Command} current command
```

Comment:
This is a loss of type information here - @johnspackman , have you consistently formatted jsdoc param type this way - then a regex search/replace could preserve the type info

Reply:
possibly, but what's wrong with the way it was before?  I thought that you were supposed to use a fully dotted classname unless the class being referred to is in the same package as this class